README.md

```diff
@@ -120,112 +106,142 @@ Exemple d'une commande test en dev sur chrome pour un fichier test particulier:
 ./pc test-cafe-pro -b chrome:headless -f signup.js
 ```
 
-### Emuler avec Cordova
 
+
+## Deploy
+
+### FRONTEND WEB
+
+Pour déployer une nouvelle version web, par exemple en staging:
+**(Attention de ne pas déployer sur la prod sans authorisation !)**
 ```bash
-yarn global add cordova-cli
+rm -rf node_modules
+yarn install
+yarn version
+./pc -e staging deploy-frontend-webapp
 ```
+Et pour pro sur staging, il suffit de remplacer la dernière commande par celle-ci:
 
 ```bash
-cd webapp && cordova run ios
+./pc -e staging deploy-frontend-pro
 ```
 
-<!--
-iPhone-5s, 11.2
-iPhone-6, 11.2
-iPhone-6-Plus, 11.2
-iPhone-6s, 11.2
-iPhone-6s-Plus, 11.2
-iPhone-7, 11.2
-iPhone-7-Plus, 11.2
-iPhone-SE, 11.2
-iPad-Air, 11.2
-iPad-Air-2, 11.2
-iPad--5th-generation-, 11.2
-iPad-Pro--12-9-inch---2nd-generation-, 11.2
-iPad-Pro--10-5-inch-, 11.2
-Apple-TV-1080p, tvOS 11.2
-Apple-TV-4K-4K, tvOS 11.2
-Apple-TV-4K-1080p, tvOS 11.2
-iPhone-8, 11.2
-iPhone-8-Plus, 11.2
-iPhone-X, 11.2
-iPad-Pro--9-7-inch-, 11.2
-iPad-Pro, 11.2
-Apple-Watch-38mm, watchOS 4.2
-Apple-Watch-42mm, watchOS 4.2
-Apple-Watch-Series-2-38mm, watchOS 4.2
-Apple-Watch-Series-2-42mm, watchOS 4.2
-Apple-Watch-Series-3-38mm, watchOS 4.2
-Apple-Watch-Series-3-42mm, watchOS 4.2
--->
+#### Publier shared sur npm
 
-### Développer en Android
+Pour publier une version de pass-culture-shared sur npm
 
-Vous pouvez utiliser une ptite config ngrok pour l'api et la webapp par exemple:
 ```bash
-cd webapp/ && yarn run ngrok
+cd shared
+npm adduser
+yarn install
+npm publish
 ```
 
-Ensuite il faut lancer l'application configurée avec ces tunnels:
+Puis sur webapp et/ou pro, mettre à jour la version de pass-culture-shared dans le fichier `package.json` :
+
 ```bash
-./pc start-browser-webapp -t
+yarn add pass-culture-shared@x.x.x 
+git add package.json yarn.lock
 ```
 
-Vous pourrez alors utiliser l'url ngrok webapp pour dans votre navigateur android.
+avec `x.x.x` nouvelle version déployée sur shared.
 
 
-## Deploy
 
-### FRONTEND WEB
 
-Pour déployer une nouvelle version web, par exemple en staging:
+### BACKEND
+
+Pour déployer une nouvelle version de l'API, par exemple en staging:
 **(Attention de ne pas déployer sur la prod sans authorisation !)**
 ```bash
-./pc -e staging deploy-frontend
+git checkout master
+git pull -r
+git tag 'v.X.Y.Z'
```

Comment:
du coup, suite aux dernières discussions, plutôt : `git tag 'vN.x'`, où `N est le numéro de l'itération et x un autoincrément qui démarre à 0 et est changé en cas de hotfix en cours d'itération`.
WDYT ?

Reply:
même message à mettre pour les yarn version sur les fronts qui n'acceptent que le semver

Reply:
yes !